conda/pyhit/meta.yaml

```diff
@@ -9,14 +9,15 @@ source:
   git_url: https://github.com/milljm/moose-pyhit.git
 
 build:
-  number: {{ build }}  # [linux,osx]
+  number:              # [linux,osx]
```

Comment:
?

Reply:
@milljm can comment on this, but we only use build indices now for petsc and libmesh

Reply:
Yes, I like removing build string where ever we can.

For all our packages which use "DATE" as the version, we don't need a build_1, build_2, build_3 string attached. `moose-libmesh` (and perhaps `moose-wasp` down the road) being the exception case(s), due to how tightly dependent those packages must remain with `moose-dev`.